spec/views/helpers/unicode_helper_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe UnicodeHelper, type: :helper do
+  describe 'clear_invisible_string' do
+    it "removes control code" do
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [Sider](https://sider.review) -->